app/Providers/AppServiceProvider.php

```diff
@@ -13,9 +13,11 @@ class AppServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    public function boot()
+    public function boot(): void
     {
         Schema::defaultStringLength(191);
+        \URL::forceScheme('https');
```

Comment:
Maybe this needs to be added with an if to see if you're in debug mode. This could possibly break development, and otherwise maybe the debug bar

Reply:
Serving without this in a remote development or production causes issues. The only usecase would be http://localhost 
Also see https://learn.microsoft.com/en-us/azure/active-directory/develop/reply-url#supported-schemes

Reply:
Well, production isn't an issue then, because that wouldn't be in debug mode :P
If this is really an issue in a remote environment we should add a second environment variable to handle that case.